easybuild/easyconfigs/r/RPostgreSQL/RPostgreSQL-0.7-6-foss-2023b.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'Bundle'
+
+name = 'RPostgreSQL'
+version = '0.7-6'
+
+homepage = 'https://cran.r-project.org/package=tesseract'
+description = """
+Database interface and 'PostgreSQL' driver for 'R'. This package provides a
+Database Interface 'DBI' compliant driver for 'R' to access 'PostgreSQL'
+database systems."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+builddependencies = [
+    ('pkgconf', '2.0.3'),
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('R', '4.3.3'),
+    ('PostgreSQL', '16.1'),
+]
+
+exts_defaultclass = 'RPackage'
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_list = [
+    ('DBI', '1.2.2', {
+        'checksums': ['8d55a0b3654511b776ae8ad201f4ebefa707c5f5f4ee6ae39af3856913d5a781'],
+    }),
```

Comment:
Rather use the R-bundle-CRAN? Might seem overkill for just one thing here, but I suspect this isn't used in isolation anyway and one would still be loading this bundle regardless.

Reply:
I will close this. I can't remember why I built it.